src/geom/detail/frameSetUtils.cc

```diff
@@ -31,10 +31,10 @@
 #include "astshim.h"
 
 #include "lsst/afw/formatters/Utils.h"
-#include "lsst/afw/coord/Coord.h"
 #include "lsst/afw/fits.h"
 #include "lsst/afw/geom/Angle.h"
 #include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/SpherePoint.h"
```

Comment:
Does this file need to be included here?

Reply:
No. Good catch.